test/fixtures/app_cicd/main.tf

```diff
@@ -47,60 +47,16 @@ module "example" {
   }
 }
 
-# VPCs
-module "vpc" {
-  for_each = var.gke_project_ids
-  source   = "terraform-google-modules/network/google"
-  version  = "~> 4.0"
-
-  project_id   = var.gke_project_ids[each.key]
-  network_name = "gke-vpc-${each.key}"
-  routing_mode = "REGIONAL"
-
-  subnets = [
-    {
-      subnet_name   = "gke-subnet"
-      subnet_ip     = "10.0.0.0/17"
-      subnet_region = var.primary_location
-    },
-  ]
-  secondary_ranges = {
-    gke-subnet = [
-      {
-        range_name    = "us-central1-01-gke-01-pods"
-        ip_cidr_range = "192.168.0.0/18"
-      },
-      {
-        range_name    = "us-central1-01-gke-01-services"
-        ip_cidr_range = "192.168.64.0/18"
-      },
-    ]
-  }
+resource "google_project_iam_member" "cluster_service_account-gcr" {
```

Comment:
Can't we `grant_registry_access` in the setup?

Reply:
Was unable to do this in setup previously since the projects were non-deterministic. I've since refactored the cluster creation back into the fixtures so I can use this again

Reply:
After refactoring into what I thought was a better form, I encountered other issues that made deployment more difficult. So for now we'll have to revert to this workaround for setup